examples/quickstart/matrix_multiplication.cpp

```diff
@@ -0,0 +1,135 @@
+{
+    std::cout << "Matrix " << message << " is:" << std::endl;
+    for (std::size_t i = 0; i < rows; i++)
+    {
+        for (std::size_t j = 0; j < cols; j++)
+            std::cout << M[i * rows + j] << " ";
+        std::cout << "\n";
+    }
+}
+//]
+
+///////////////////////////////////////////////////////////////////////////////
+//[mul_hpx_main
+int hpx_main(hpx::program_options::variables_map& vm)
+{
+    using element_type = int;
+
+    // Define matrix sizes
+    std::size_t rowsA = vm["n"].as<std::size_t>();
+    std::size_t colsA = vm["m"].as<std::size_t>();
+    std::size_t rowsB = colsA;
+    std::size_t colsB = vm["k"].as<std::size_t>();
+    std::size_t rowsR = rowsA;
+    std::size_t colsR = colsB;
+
+    // Initialize matrices A and B
+    std::vector<int> A(rowsA * colsA);
+    std::vector<int> B(rowsB * colsB);
+    std::vector<int> R(rowsR * colsR);
+
+    // Define seed
+    unsigned int seed = std::random_device{}();
+    if (vm.count("seed"))
+        seed = vm["seed"].as<unsigned int>();
+
+    gen.seed(seed);
+    std::cout << "using seed: " << seed << std::endl;
+
+    // Define range of values
+    std::size_t lower = vm["l"].as<std::size_t>();
+    std::size_t upper = vm["u"].as<std::size_t>();
+
+    // Matrices have random values in the range [lower, upper]
+    std::uniform_int_distribution<element_type> dis(lower, upper);
+    auto generator = std::bind(dis, gen);
+    hpx::ranges::generate(A, generator);
+    hpx::ranges::generate(B, generator);
+
+    // Perform matrix multiplication
+    hpx::for_loop(hpx::execution::par, 0, rowsA, [&](auto i) {
+        hpx::for_loop(0, colsB, [&](auto j) {
+            R[i * rowsR + j] = 0;
+            hpx::for_loop(0, rowsB, [&](auto k) {
+                R[i * rowsR + j] += A[i * rowsA + k] * B[k * rowsB + j];
+            });
+        });
+    });
+
+    // Print all 3 matrices
+    std::cout << "Matrix A is : \n";
+    for (std::size_t i = 0; i < rowsA; i++) {
+        for (std::size_t j = 0; j < colsA; j++)
+            std::cout << A[i * colsA + j] << " ";
+        std::cout << "\n";
+    }
+
+    print_matrix(A, rowsA, colsA, "A");
+    std::cout << "\n";
+    print_matrix(B, rowsB, colsB, "B");
+    std::cout << "\n";
+    print_matrix(R, rowsR, colsR, "R");
+    std::cout << "\n";
+
+    return hpx::local::finalize();
+}
+//]
+
+///////////////////////////////////////////////////////////////////////////////
+//[mul_main
+int main(int argc, char* argv[])
+{
+    using namespace hpx::program_options;
+    options_description cmdline("usage: " HPX_APPLICATION_STRING " [options]");
+    // clang-format off
+    cmdline.add_options()
+        ("n",
+        hpx::program_options::value<std::size_t>()->default_value(2),
+        "Number of rows of first matrix")
+        ("m",
+        hpx::program_options::value<std::size_t>()->default_value(3),
+        "Number of columns of first matrix (equal to the number of rows of second matrix)")
```

Comment:
This line should be shorter... Since it's inside `// clang-format off` you'll have to manually wrap it to 80 characters.